### DISCUSSION TITLE
The roaster can"t connect with laptop

### DISCUSSION THREAD

**Discussion by FlypDexter** [artisan-roaster-scope/artisan#675] [Q&A]
Hi, can i know how to connect the roasters with laptop, the artisan software done installed, the port also done

**Comment by CarefreeBuzzBuzz** [comment]
Sorry but without any details no help can be provided.  Please read through the section of the Quick Start Guide (find it where you got the software) on the setup and then come back with detailed questions.  

___________________________
Michael Herbert, CarefreeBuzzBuzz
